src/engine/enginexfader.cpp

```diff
@@ -51,12 +51,17 @@ void EngineXfader::getXfadeGains(
         *gain2 = 0.0;
     }
 
-    if (constPower) {
+    if (curve == MIXXX_XFADER_CONSTPWR) {
         if (*gain1 > *gain2) {
             *gain2 = 1 - *gain1;
         } else {
             *gain1 = 1 - *gain2;
         }
+
+        // normalize the gain
```

Comment:
It would be grate if you can put here the essence of our discussion. 
Here a first draft: 
```
// The resulting power at the crossover point depends on the correlation of the resulting signals
// In theory the gain ration varies from 0.5 for two equal signals to sqrt(0.5) (0.707) for totally 
// uncorrelated signals. 
// Since the underlying requirement for this curve is constant loudness, we did a test with 30 s
// snippets of various genres and ReplayGain 2.0 analysis. Allmost all results where near 0.707 
// with one exception Mixing two parts of the same track, which results to 0.66  
// Details:
// ..
```